### PR TITLE
daml-assistant: Add `daml sandbox-next`.

### LINKED ISSUE #4085: Integrate Sandbox-on-X with DAML Assistant
The new runner should be available in the SDK under a new (and temporary) DAML Assistant command `sandbox-next`. This allows users to easily try out the Sandbox on the new architecture and provide feedback.

Depends on #4084.

### PR DESCRIPTION
**Pull request by SamirTalwar-DA**
### Changelog

- [DAML Assistant] You can now run a pre-release version of Sandbox with `daml sandbox-next` so you can test it out and verify everything is working as expected. Running this will launch Sandbox rebuilt on a more modern architecture. An upcoming release of DAML will switch over to the new implementation by default.

Resolves #4085.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: release/sdk-config.yaml.tmpl [hunk 1]
```diff
@@ -43,6 +43,10 @@ commands:
   path: daml-helper/daml-helper
   desc: "Launch the Sandbox"
   args: ["run-jar", "--logback-config=daml-sdk/sandbox-logback.xml", "daml-sdk/daml-sdk.jar", "sandbox"]
+- name: sandbox-next
+  path: daml-helper/daml-helper
+  desc: "Launch the Sandbox's upcoming implementation (for testing purposes)"
```
Comment (cocreature) [inline release/sdk-config.yaml.tmpl]:
```suggestion
  desc: "Launch the Sandbox's upcoming implementation (experimental, for testing purposes)"
```
We use that for other features (e.g. triggers) so given that this also seems fairly experimental at this stage it seems helpful to make that explicit.

**Comment by SamirTalwar-DA** [comment]
Regarding integration tests, yes, good point. I forgot but I can add one.

**Comment by SamirTalwar-DA** [comment]
We do have one main limitation: scenarios aren't supported. We advise people to use DAML Script instead. Otherwise, I _think_ everything should just work.

**Comment by SamirTalwar-DA** [comment]
@bame-da: Should we write a blog post about this?

**Pull request merged**